.gitignore

```diff
@@ -17,3 +17,10 @@ project.lock.json
 /.dotnet/
 tools/**/obj/
 tools/**/build/
+
+# Docs artifacts
+docs/obj/
+docs/bin/
```

Comment:
Maybe we should just ignore obj and bin directories everywhere, we've got 4 sets of these patterns now.